maintenance/health/servicehealthcheck/healthchecker.go

```diff
@@ -0,0 +1,123 @@
+// Copyright © 2019 by PACE Telematics GmbH. All rights reserved.
+// Created at 2019/10/18 Charlotte Pröller
+
+package servicehealthcheck
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/pace/bricks/maintenance/log"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+)
+
+type HealthCheck interface {
+	Name() string
+	InitHealthCheck() error
+	HealthCheck(currTime time.Time) (bool, error)
```

Comment:
Why is it passing the current time? It's not like the current time can be anything else than `time.Now()`, logically. It also implies that a health check is doing throttling, which should be completely the responsibility of the health check. Or worse, it could imply a health check should be able to pull up results from the past, i.e. provide a full history.